syntax_and_semantics/visibility.md

```diff
@@ -118,7 +118,7 @@ end
 Namespace::Bar.new.bar
 ```
 
-A `protected` class method can be invoked from an instance method and the other way around:
+A `protected` class method can be invoked from an instance method (of the same class) and the other way around:
```

Comment:
Adding parenthesis is unnecessary.

Reply:
Agreed. It seemed unnecessary to go back and change it however since there might also be a better way of wording the "and the other way around" part.

Something like: "A `protected` class or instance method can be invoked from any other one of the same class.